src/main/java/io/cdap/plugin/salesforce/parser/SalesforceQueryVisitor.java

```diff
@@ -64,7 +64,7 @@ public SObjectDescriptor visitStatement(SOQLParser.StatementContext ctx) {
 
       if (!aliasedFields.isEmpty()) {
         throw new SOQLParsingException("Field aliasing is not allowed except when using a GROUP BY clause. "
-          + "List of fields with alias: " + aliasedFields);
+                                         + "List of fields with alias: " + aliasedFields);
```

Comment:
What is the context for the changes here ? Is it only formatting? Skip formatting for the code that is not changed for this PR. Hard to see the exact changes otherwise.

Reply:
these are just format issues.